Realm.xcodeproj/xcshareddata/xcschemes/TestHost.xcscheme

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
No, but there's a long twitter thread where others confirm this as an issue here: https://twitter.com/simjp/status/665250128656142337